app/src/main/java/com/io1/todo/feature/onboarding/utils/OnBoardingPage.kt

```diff
@@ -0,0 +1,29 @@
+package com.io1.todo.feature.onboarding.utils
+
+import androidx.annotation.DrawableRes
+import com.io1.todo.R
+
+sealed class OnBoardingPage(
+    @DrawableRes
+    val image: Int,
+    val title: String,
+    val description: String
+) {
+    object First : OnBoardingPage(
+        image = R.drawable.first,
+        title = "Meeting",
```

Comment:
I would suggest that all strings should be in strings.xml, call them with string resources